programs/uxd/src/instructions/redeem.rs

```diff
@@ -149,6 +155,9 @@ pub struct Redeem<'info> {
 
     /// #21
     pub rent: Sysvar<'info, Rent>,
+
+    /// #22
+    pub clock: Sysvar<'info, Clock>,
```

Comment:
clock could be accessed using `Clock::get()` I think, so not necessary to pass the this account